standalone/ansible/roles/postgres/templates/pg_backup.sh

```diff
@@ -19,4 +19,4 @@ else
   {% endfor %}
 fi
 
-find $BACKUP_DIR -maxdepth 0 -mtime +$DAYS_TO_KEEP -exec rm -rf '{}' ';'
+find $BACKUP_DIR -maxdepth 1 -mtime +$DAYS_TO_KEEP -exec rm -rf '{}' ';'
```

Comment:
This is the actual problem that was preventing the backups from being removed?

Reply:
Yeah, `-maxdepth 0` does something different altogether. https://unix.stackexchange.com/questions/162411/find-maxdepth-0-not-returning-me-any-output